wazo_webhookd/plugins/subscription/api.yml

```diff
@@ -177,6 +177,21 @@ paths:
           $ref: '#/responses/NotFoundError'
         '503':
           $ref: '#/responses/AnotherServiceUnavailable'
+  /subscriptions/{subscription_uuid}/hook-log:
+      summary: Get hook results
+      description: '**Required ACL:** `webhookd.subscriptions.{subscription_uuid}.hook_log`'
```

Comment:
the ACL should be suffixed by .read for a GET the implementation also uses the plural form

`webhookd.subscriptions.{subscription_uuid}.hook_logs.read`

Reply:
done